### PR TITLE
[Bugfix:TAGrading] Show hidden points row to grader

### PR DESCRIPTION
**Pull request by loadedskiffer**
### What is the current behavior?
fixes issue #6667 Currently when a grader is grading an assignment the autograding panel does not show the Autograding total with Hidden points unless the grades have been released.

### What is the new behavior?
All Graders EXCEPT PEER GRADERS can now see the row for autograding total with hidden points while grading. Students still can't see hidden points until grades are released.

### Other information?
To test:
Grade 2 assignments with hidden auto grading points, one where grading is still open and one where grades have been released, see that you can see the hidden points row no matter what.

### CODE REVIEW

**Review by eddiekrystowski — APPROVED** [review]
Looks good to me, works as described.

File: site/app/views/AutoGradingView.php [hunk 1]
```diff
@@ -83,8 +83,13 @@ public function showResults(AutoGradedVersion $version_instance, bool $show_hidd
             }
 
             $show_hidden_breakdown = $any_visible_hidden && $show_hidden;
+            //always show hidden points to the grader
+            if ($ta_grading === true) {
+                $show_hidden_breakdown = true;
+            }
```
Comment (MasterOdin) [inline site/app/views/AutoGradingView.php:89]:
```suggestion
            $show_hidden_breakdown = $ta_grading || ($any_visible_hidden && $show_hidden);
```

File: site/app/views/grading/ElectronicGraderView.php [hunk 2]
```diff
@@ -899,6 +899,12 @@ public function hwGradingPage(Gradeable $gradeable, GradedGradeable $graded_grad
         $return = "";
         $is_notebook = $gradeable->getAutogradingConfig()->isNotebookGradeable();
 
+        //$ta_grading is used in AutoGradingView to determine if hidden autograding points will be shown, we want to always show them to graders unless they are peer graders
+        $ta_grading = true;
+        if ($this->core->getUser()->getGroup() == User::GROUP_STUDENT) {
+            $ta_grading = false;
+        }
```
Comment (MasterOdin) [inline site/app/views/grading/ElectronicGraderView.php:906]:
```suggestion
        $ta_grading = $this->core->getUser()->getGroup() !== User::GROUP_STUDENT;
```

**Comment by MasterOdin** [comment]
@loadedskiffer, just want to confirm that not showing the hidden grading totals to peers is desired behavior has been discussed @bmcutler.

**Comment by loadedskiffer** [comment]
> @loadedskiffer, just want to confirm that not showing the hidden grading totals to peers is desired behavior has been discussed @bmcutler.

@MasterOdin Yes, in pr #5982, we decided to make it so that peer graders see nothing about autograding points on the rubric panel and don't see hidden points on the autograding panel.

**Pull request merged**